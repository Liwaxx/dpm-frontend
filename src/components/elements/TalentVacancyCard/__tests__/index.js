import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../index';

jest.mock('../../../elements/Text');

describe('TalentVacancyCard', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Component data={{}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders with custom class', () => {
    const tree = renderer
      .create(<Component data={{}} className="test" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});