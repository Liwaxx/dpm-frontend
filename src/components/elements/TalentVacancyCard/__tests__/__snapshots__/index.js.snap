// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TalentVacancyCard renders correctly 1`] = `
<div
  className="Component-root-11"
>
  <div
    className="Component-content-1"
  >
    <div
      className="Component-contentItem-2"
    >
      <div
        className="Component-lineContainer-6"
      >
        <div
          className="Component-line-5"
        >
          <div
            className="Component-lineGrey-8"
            style={
              Object {
                "flex": "NaN 1 auto",
              }
            }
          />
          <div
            className="Component-lineRed-9"
            style={
              Object {
                "flex": "undefined 1 auto",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="Component-contentItem-2"
    >
      <div
        className="Component-lineContainer-6"
      >
        <div
          className="Component-line-5"
        >
          <div
            className="Component-lineGreen-7"
            style={
              Object {
                "flex": "undefined 1 auto",
              }
            }
          />
          <div
            className="Component-lineGrey-8"
            style={
              Object {
                "flex": "NaN 1 auto",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`TalentVacancyCard renders with custom class 1`] = `
<div
  className="Component-root-11 test"
>
  <div
    className="Component-content-1"
  >
    <div
      className="Component-contentItem-2"
    >
      <div
        className="Component-lineContainer-6"
      >
        <div
          className="Component-line-5"
        >
          <div
            className="Component-lineGrey-8"
            style={
              Object {
                "flex": "NaN 1 auto",
              }
            }
          />
          <div
            className="Component-lineRed-9"
            style={
              Object {
                "flex": "undefined 1 auto",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="Component-contentItem-2"
    >
      <div
        className="Component-lineContainer-6"
      >
        <div
          className="Component-line-5"
        >
          <div
            className="Component-lineGreen-7"
            style={
              Object {
                "flex": "undefined 1 auto",
              }
            }
          />
          <div
            className="Component-lineGrey-8"
            style={
              Object {
                "flex": "NaN 1 auto",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
